### ISSUE TITLE
selecting more than one user role

### ISSUE THREAD

**Issue by tokr-coder** [MrEbabi/New-Order-Notification-for-Woocommerce-Plugin#3]
I'm testing your latest version and discovered a bug.
When I select multiple user roles to allow access to the notifications page ( for example: admin + shop manager ), as admin I am no longer able to access the notifications page. If only one role is selected (i.e. admin), then access is permitted.

PS. It seems to me that the way the list of permitted roles is updated is not intuitive. If I have admin in the list and I select Editor in the selector box and save, then Editor replaces Admin in the list. But I was expecting Editor to be added to the end of the list. 
Would it be possible to remove the multi-select box and keep just the list of roles with checkboxes so that roles can be checked and unchecked as needed? The default could be all roles checked or maybe just admin checked.

**Comment by MrEbabi** [comment]
Hello,

Thanks for warning me up. I got lots of emails and bugs reports for v1.3.0 and v1.3.1. Just released v1.3.3 to fix all bugs and warnings.

It will be very good if you can check the newest version and mention if  you face any problems. Thanks for your effort and contributions!

**Comment by tokr-coder** [comment]
Hi, thanks for updating,
The first thing I encountered was that the new version overwrote the settings of the previous version, which caused some problems on the customer's site when the plugin auto-updated. But we figured that out quickly.

The role permission selection system still doesn't work well.
When I add roles until all the roles are selected, the checkbox list on the right vanishes. If I then add another role with the multiselect box, the list re-appears and there is a duplicate role.

I suggest:
Remove the multiselect box. replace it with a list of checkboxes (like the Notification Order Statuses). They will appear checked or unchecked when the role is permitted or not permitted. Then you won't need the list on the right with the Permitted User Roles or the Remove User Role button.

The UX of the product ID selection is also not great when there are a lot of products.
I have 100 products. If I select all in the product IDs box and then save, the Alert for Products with IDs list remains empty. So I then select one product and save, now I have 101 product IDs in the list (and the Product IDs box is empty). 
I don't like that the box has product names and the list is a list of ID's.
And there needs to be a "select all" button! Now I have the check the box on 101 products to remove them.

I'm not sure what would be a better way of managing the product IDs. Maybe something like the Upsells and Crossells selection/search system in woocommerce? Maybe selecting categories or tags instead of products?
